ctakesclient/client.py

```diff
@@ -20,26 +20,27 @@ def get_url_ctakes_rest() -> str:
 
     :return: URL_CTAKES_REST env variable or default using localhost
     """
-    return os.environ.get(
-        'URL_CTAKES_REST',
-        'http://localhost:8080/ctakes-web-rest/service/analyze')
+    url = os.environ.get('URL_CTAKES_REST')
+    return url or 'http://localhost:8080/ctakes-web-rest/service/analyze'
 
 
-def post(sentence: str, url=get_url_ctakes_rest()) -> dict:
+def post(sentence: str, url: str = None) -> dict:
     """
     :param sentence: clinical text to send to cTAKES
     :param url: cTAKES REST server fully qualified path
     :return:
     """
+    url = url or get_url_ctakes_rest()
```

Comment:
This change was made for testability (`get_url_ctakes_rest` gets called every time, rather than once at import time) and convention -- you don't usually see method calls in kwarg definitions because of the risk of side effects.